roles/pulp-api/tasks/signing_service.yml

```diff
@@ -11,6 +11,12 @@
         status: "False"
         lastTransitionTime: "{{ lookup('pipe', 'date --iso-8601=seconds') }}"
 
+- name: Set GPG initContainer image
```

Comment:
I just wanted to note that in the future, we should probably make this init container overridable by the user as a CRD param for community deployments with air-gapped deployments. But not a requirement for merge here.